src/modules/users/components/input-fields/first-name-input.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef } from 'react';
+import Box from '@mui/material/Box';
+import { BasicInput } from '#/modules/users/components';
+import { type FormInputProps } from '../../types';
+import { isInputFieldEmpty } from '#/utils';
+import { inputStyles as styles } from '../../styles/input.styles';
+
+export function FirstNameInput(props: FormInputProps) {
+  const { focusedField } = props;
+  const firstNameRef = useRef<HTMLInputElement>();
+
+  useEffect(() => {
+    firstNameRef.current?.focus();
+  }, []);
```

Comment:
Why do we need this? I don't see you added it in other input components.

Reply:
it's a little hack to trigger the focus of custom input fields to show populated user information